modules/@apostrophecms/ui/ui/apos/components/AposSlat.vue

```diff
@@ -1,110 +1,117 @@
 
 <template>
-  <transition name="fade">
```

Comment:
transition removed?

Reply:
Eslint error, not sure it worked at all:
```
The element inside `<transition>` is expected to have a `v-if` or `v-show` directive. 3:5:71 eslint vue/require-toggle-inside-transition
```